### ISSUE TITLE
BTC-BSQ: java.lang.NullPointerException

### ISSUE THREAD

**Issue by mattmill30** [bisq-network/bisq#4219]
### Description

java.lang.NullPointerException encountered when accessing 'Portfolio > Open Trades', after confirming a BTC-BSQ sell offer.

Error wasn't encountered during take offer, process, but after clicking 'Go to Portfolio > Open Trades' on trade confirmation pop-up.

#### Version

v1.3.2

### Screenshots

![image](https://user-images.githubusercontent.com/6415621/80765598-5397a580-8b3b-11ea-97d0-7f12fa34a65d.png)

#### Device or machine

Win10 x64

#### Additional info
[bisq-BSQ_NullPointerException.log](https://github.com/bisq-network/bisq/files/4561723/bisq-BSQ_NullPointerException.log)

**Comment by sqrrm** [comment]
From log

```
Apr-30 23:13:30.235 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: Uncaught Exception from thread JavaFX Application Thread 
Apr-30 23:13:30.236 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: throwableMessage= null 
Apr-30 23:13:30.236 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: throwableClass= class java.lang.NullPointerException 
Apr-30 23:13:30.563 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: Stack trace:
java.lang.NullPointerException
        at javafx.controls/javafx.scene.control.Tooltip$TooltipBehavior.lambda$new$0(Tooltip.java:932)
        at javafx.graphics/javafx.animation.Animation.finished(Animation.java:1131)
        at javafx.graphics/javafx.animation.AnimationAccessorImpl.finished(AnimationAccessorImpl.java:49)
        at javafx.graphics/com.sun.scenario.animation.shared.SingleLoopClipEnvelope.timePulse(SingleLoopClipEnvelope.java:103)
        at javafx.graphics/javafx.animation.Animation.doTimePulse(Animation.java:1101)
        at javafx.graphics/javafx.animation.Animation$1.lambda$timePulse$0(Animation.java:186)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at javafx.graphics/javafx.animation.Animation$1.timePulse(Animation.java:185)
        at javafx.graphics/com.sun.scenario.animation.AbstractMasterTimer.timePulseImpl(AbstractMasterTimer.java:344)
        at javafx.graphics/com.sun.scenario.animation.AbstractMasterTimer$MainLoop.run(AbstractMasterTimer.java:267)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:514)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:498)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:491)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:319)
        at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:175)
        at java.base/java.lang.Thread.run(Thread.java:844)
```

**Comment by ripcurlx** [comment]
Seems to be some JavaFX issue. Hopefully when we are able to upgrade Java and JavaFX most of the issues should be gone. 🤞

**Comment by ripcurlx** [comment]
Should be gone as soon as #4242 gets merged.

**Comment by ripcurlx** [comment]
Closing as complete as #4242 was merged already with v1.6.5.
